oracledb-ballerina/tests/constants.bal

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/file;
+import ballerina/sql;
+
+const string user="admin";
+const string password="password";
+const string host = "localhost";
+const int port = 1521;
+const string database = "ORCLCDB.localdomain";
```

Comment:
Let's uppercase all the constants

Reply:
changed with 171baaf97be39b48c6663ec8edeb3d05944f98f0